boards/trex-one.json

```diff
@@ -0,0 +1,42 @@
+{
+  "build": {
+    "core": "arduino",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_RASPBERRY_PI_PICO -DARDUINO_ARCH_RP2040 -DARDUINO_TREX_ONE",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x1209",
+        "0xD00F"
+      ]
+    ],
+    "mcu": "rp2040",
+    "variant": "TREX_ONE"
+  },
+  "debug": {
+    "jlink_device": "RP2040_M0_0",
+    "openocd_target": "rp2040.cfg",
+    "svd_path": "rp2040.svd"
+  },
+  "frameworks": [
+    "arduino"
+  ],
+  "name": "TREX One (RP2040)",
+  "upload": {
+    "maximum_ram_size": 270336,
+    "maximum_size": 16384000,
```

Comment:
I believe 16MB is external flash? How will people be able to utilize it?

Reply:
The RP2040 has no internal flash. The RP2040 Pico Board had 4MB of external flash and the TREX One has 16MB. With FreeRTOS, TinyUSB, LVGL etc. 16MB can be very practical (think of LittleFS / SPIFFS).